### ISSUE TITLE
mu-debug fails to start - Linux, Windows, Raspberry Pi - is it still needed?

### ISSUE THREAD

**Issue by martinohanlon** [mu-editor/mu#353]
When you run `mu-debug` it fails to start raising the error:

```
C:\Users\Martin O'Hanlon-LT>mu-debug
Traceback (most recent call last):
  File "c:\users\martin o'hanlon-lt\appdata\local\programs\python\python36\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\users\martin o'hanlon-lt\appdata\local\programs\python\python36\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\Martin O'Hanlon-LT\AppData\Local\Programs\Python\Python36\Scripts\mu-debug.exe\__main__.py", line 9, in <module>
  File "c:\users\martin o'hanlon-lt\appdata\local\programs\python\python36\lib\site-packages\mu\app.py", line 144, in debug
    filename = os.path.normcase(os.path.abspath(sys.argv[1]))
IndexError: list index out of range
```

![image](https://user-images.githubusercontent.com/2329501/36780133-fff911e4-1c69-11e8-87bb-67b074eab75b.png)

This occurs on Windows, Linux and Raspberry Pi.

Is `mu-debug` still needed? It looks like it was created to launch the debugger around the beta8 stage, but the debugger runs as default now

**Comment by carlosperate** [comment]
`mu-debug` is used internally to start the debugger process when the debugger view is opened, the app launches `mu-debug` with the script path and other flags.

This is probably related to https://github.com/mu-editor/mu/issues/346.

**Comment by martinohanlon** [comment]
I understand now, that mu-debug is not a command to be run by the users, its an internal entry point.

**Comment by carlosperate** [comment]
Btw, the error was a missing command line arguments, so not related to #_346.

@ntoll maybe there could be a simple print and exit when no arguments are given. Users could accidentally run the debugger instead of mu and this could be an intimidating error message.

**Comment by ntoll** [comment]
Good call. I'll add a fix.

**Comment by ntoll** [comment]
Fixed by 5c5d92c477346478a9df82065cfeb881ffd21dd2 ;-)
